doc/nrf/app_dev/bootloaders_and_dfu/fw_update.rst

```diff
@@ -295,9 +295,9 @@ Software-based downgrade prevention
 The |NCS| supports MCUboot's software-based downgrade prevention for application images, using semantic versioning.
 This feature offers protection against any outdated firmware that is uploaded to a device.
 
-You can enable this feature by setting the configuration options ``CONFIG_MCUBOOT_DOWNGRADE_PREVENTION`` and ``CONFIG_BOOT_UPGRADE_ONLY`` for the MCUboot image.
+To enable this feature, set the configuration options ``CONFIG_MCUBOOT_DOWNGRADE_PREVENTION`` and ``CONFIG_BOOT_UPGRADE_ONLY`` for the MCUboot image.
```

Comment:
Are these two Kconfig options correct? At least they are not found in the reference documentation.

Reply:
See note above